src/DataXml/Utd820/CommonXmlNodes.cs

```diff
@@ -0,0 +1,705 @@
+namespace Diadoc.Api.DataXml.Utd820
```

Comment:
Maybe common xml nodes should not be in `Utd820` namespace?
I think `Diadoc.Api.DataXml` is enough for common node types here.
At your discretion

Reply:
It's interesting question. I was thinking where to place them. Both ways are possible though. 

Moved to `Diadoc.Api.DataXml`